python/GafferUI/CompoundEditor.py

```diff
@@ -1661,8 +1663,35 @@ def getToolTip( self ) :
 			for d in drivenEditors :
 				toolTipElements.append( " - _%s_" % d.getTitle() )
 
+		self.__highlightRelatedEditors( editor, True )
```

Comment:
Outrageous! I suppose it's highly unlikely that anyone will trigger chaos by calling `getToolTip()` directly though, since this is a private widget squirrelled way down in the hierarchy. I think it warrants some sort of Hail Mary comment though...

Reply:
Added disclaimer in https://github.com/GafferHQ/gaffer/pull/3360/commits/08163114dcd10e1c195d06d7e93e25062af0e8a0